ERCS/erc-7662.md

```diff
@@ -0,0 +1,273 @@
+---
+eip: 7662
+title: AI Agent NFTs
+description: A specification for NFTs that represent AI Agents.
+author: Greg Marlin (@marleymarl)
+discussions-to: https://ethereum-magicians.org/t/erc-7662-ai-agent-nfts/19371
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-03-26
+requires: 721
+---
+
+## Abstract
+
+This proposal introduces a standard for AI agent NFTs. In order for AI Agents to be created and traded as NFTs it doesn't make sense to put the prompts in the token metadata, therefore it requires a standard custom struct. It also needs doesn't make sense to store the prompts directly onchain as they can be quite large, therefore this standard proposes they be stored as decentralized storage URLs. This standard also proposes two options on how this data should be made private to the owner of the NFT, with the favored implementation option being encrypting the data using custom contract parameters for decryption that decrypt only to the owner of the NFT. 
```

Comment:
Your abstract does a good job of covering a technical overview, but it doesn't really explain the goal of the proposal very well. After reading the abstract, the reader should have _both_ a technical overview and an idea of the problem the proposal is trying to solve.

Reply:
Got it, will add that, thanks again.